docker/tests/containerTestCommon.psm1

```diff
@@ -0,0 +1,202 @@
+$script:forcePull = $true
+# Get docker Engine OS
+function Get-DockerEngineOs
+{
+    docker info --format '{{ .OperatingSystem }}'
+}
+
+# Call Docker with appropriate result checksfunction Invoke-Docker 
+function Invoke-Docker
+{
+    param(
+        [Parameter(Mandatory=$true)]
+        [string[]]
+        $Command,
+        [ValidateSet("error","warning",'ignore')]
+        $FailureAction = 'error',
+
+        [Parameter(Mandatory=$true)]
+        [string[]]
+        $Params,
+
+        [switch]
+        $PassThru,
+        [switch]
+        $SupressHostOutput
+    )
+
+    $ErrorActionPreference = 'Continue'
+
+    # Log how we are running docker for troubleshooting issues
+    Write-Verbose "Running docker $command $params" -Verbose
+    if($SupressHostOutput.IsPresent)
+    {
+        $result = &'docker' $command $params 2>&1
```

Comment:
the construct `& 'docker'` is the same as `docker`, i'm not sure what the extra `& '...` provides